src/auro-checkbox-group.js

```diff
@@ -242,7 +242,9 @@ export class AuroCheckboxGroup extends LitElement {
         el.removeAttribute('error');
       }
 
-      el.disabled = this.disabled;
+      if (this.disabled) {
+        el.setAttribute('disabled', '');
+      }
```

Comment:
If the intention is to enable the element when `this.disabled` is `false`, we need code that would remove the `disabled` attribute in that case? Is that handled elsewhere that I am not seeing here? 

Otherwise, we need something like this maybe? 

```js
if (this.disabled) {
  el.setAttribute('disabled', '');
} else {
  el.removeAttribute('disabled');
}
```

Also, there should be a new test to support this change.

Reply:
The issue with this is that the current workflow, this function runs upon load and we support individual checkboxes having the `disabled` attribute. So with your proposed change, the individual checkboxes will have their `disabled` attribute removed when the checkbox-group does not have the `disabled` attribute.